repo/packages/A/artifactory-secondary/2/resource.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images": {
+    "icon-small": "https://raw.githubusercontent.com/jainishshah17/Images/master/jfrog_sml.png",
+    "icon-medium": "https://raw.githubusercontent.com/jainishshah17/Images/master/jfrog_med.png",
+    "icon-large": "https://raw.githubusercontent.com/jainishshah17/Images/master/jfrog_lrg.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "artifactory-pro": "jfrog-docker-reg2.bintray.io/jfrog/artifactory-pro:4.8.0",
```

Comment:
Can we remove the `artifactory-pro` image here? I believe this package is only relevant for the HA version right?